frontends/election-manager/src/components/smartcard_modal/program_election_card_view.tsx

```diff
@@ -61,31 +75,42 @@ export function ProgramElectionCardView({
   }
 
   return (
-    <Prose textCenter>
-      <h2>Program Election Card</h2>
-      {/* An empty div to maintain space between the header and subsequent p. TODO: Consider adding
-        a `maintainSpaceBelowHeaders` prop to `Prose` */}
-      <div />
-      {actionStatus && <StatusMessage actionStatus={actionStatus} />}
+    <Prose textCenter theme={fontSizeTheme.medium}>
+      {actionStatus && (
+        <p>
+          <StatusMessage actionStatus={actionStatus} />
+        </p>
+      )}
+      <Heading marginTop={showingSuccessOrErrorMessage ? '1em' : '0'}>
+        Create New Election Card
+      </Heading>
+      {electionDefinition && (
+        <p>{electionToDisplayString(electionDefinition.election)}</p>
+      )}
+
+      <HorizontalRule />
       {electionDefinition ? (
-        <p>Admin and Poll Worker cards only work for the current election.</p>
+        <p>
+          <Button disabled={!electionDefinition} onPress={programAdminCard}>
+            Admin Card
```

Comment:
Maybe change to "Create Admin Card" to match the language of the other buttons (which all have a verb)

Reply:
@jonahkagan - With the title of this fullscreen modal being "Create New Election Card", I was treating the buttons as choices of what type of card to create. The title is not a full question, but felt clear enough. Having "Create" next to each type seemed a bit verbose.

Reply:
Chatted with @beausmith and we're going to keep this as is for now

Down the road, the screen real estate piece will become less of an issue as we lock cards down to the user roles printed on them and will only need to display one option. We can add "Create" back in, then